packages/example-app-base/package.json

```diff
@@ -46,6 +46,7 @@
     "@microsoft/load-themed-styles": "^1.7.13",
     "@uifabric/example-data": "^7.0.1",
     "@uifabric/fluent-theme": "^7.1.4",
+    "@uifabric/react-hooks": "^7.0.1",
```

Comment:
@KevinTCoughlin - do you think there's any implications on any of your consumption of this package?

Reply:
example-app-base should only be referenced at development time (also tsx-editor already depends on react-hooks), so it shouldn't have an impact on production bundles.